wire-runtime/build.gradle

```diff
@@ -95,16 +86,4 @@ repositories.whenObjectAdded {
 }
 
 apply from: 'jvm.gradle'
-apply from: "$rootDir/gradle/gradle-mvn-mpp-push.gradle"
-
-publishing {
-  // Use default artifact name for the JVM target
```

Comment:
Make sure this config doesn't get lost: by default the publishing plugin will apply the `wire-runtime` name to the common artifact, we're overriding it for backwards compat reasons, so that the JVM artifact is still called `wire-runtime`.